cmd/deviceshifu/cmdtcp/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"github.com/edgenesis/shifu/pkg/logger"
+	"os"
+
+	"github.com/edgenesis/shifu/pkg/deviceshifu/deviceshifubase"
+	"github.com/edgenesis/shifu/pkg/deviceshifu/deviceshifutcp"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+func main() {
+	deviceName := os.Getenv("EDGEDEVICE_NAME")
+	namespace := os.Getenv("EDGEDEVICE_NAMESPACE")
+
+	deviceShifuMetadata := &deviceshifubase.DeviceShifuMetaData{
+		Name:           deviceName,
+		ConfigFilePath: deviceshifubase.DeviceConfigmapFolderPath,
+		KubeConfigPath: deviceshifubase.KubernetesConfigDefault,
+		Namespace:      namespace,
+	}
+
+	ds, err := deviceshifutcp.New(deviceShifuMetadata)
+	if err != nil {
+		panic(err.Error())
```

Comment:
panic or log the error first?

Reply:
I write it according to other Deviceshifu like opcua [here](https://github.com/Edgenesis/shifu/blob/60da51054a230fe6ae47436c0cceb96730484cdc/cmd/deviceshifu/cmdopcua/main.go#L26), do I need to modify all other Deviceshifu to add log.Error or only modify this and leave the inconsistency?

Reply:
ok, let's not change for now